src/Installers/Windows/Wix.props

```diff
@@ -25,6 +25,7 @@
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);files=$(MSBuildThisFileDirectory)files</DefineConstants>
     <DefineConstants>$(DefineConstants);Culture=$(Cultures)</DefineConstants>
+    <GenerateNupkgPowershellScript>$(RepoRoot)\src\Installers\Windows\GenerateNugetPack.ps1</GenerateNupkgPowershellScript>
```

Comment:
Where is this copy of the property used?

Reply:
https://github.com/aspnet/AspNetCore/pull/11345/files#diff-d09a1ae45b24853c16c713e250339fd3R89